src/components/layout/DeployRegistration/DeployRegistration.tsx

```diff
@@ -0,0 +1,40 @@
+import logo from '../../../../assets/images/logo/looped-winston-white.gif';
+import { RegistrationProgress } from '../../inputs/progress';
+
+function DeployRegistration() {
+  return (
+    <>
+      <div className="flex-column center">
+        <RegistrationProgress
+          stages={{
+            1: {
+              title: 'Pick Domain',
+              status: '',
+            },
+            2: {
+              title: 'Create an ANT',
+              status: '',
+            },
+            3: {
+              title: 'Register Name',
+              status: '',
+            },
+            4: {
+              title: 'Complete',
+              status: '',
```

Comment:
we're achieving this through the ordering of keys, but it might be good to have `dependsOn: [1, 2]`  and `next` explicitly defined in the object, so we clearly understand the dependencies of each step. not necessary for this PR, but something to keep in mind

Reply:
added todo